### DISCUSSION TITLE
Paperless as a One-Click Deployment

### DISCUSSION THREAD

**Discussion by SpuriousGer** [coollabsio/coolify#2349] [Service Template Requests]
Paperless is a document management system for PDFs (and emails, Word docs, other office documents).
It offers multiple Docker Compose standard files, see here: https://github.com/paperless-ngx/paperless-ngx/tree/main/docker/compose

There are a few aspects that would make it more complex:
- Dedicated storage volumes on the server (e.g. SMB shares)
- Potentially add auto-backup for those

**Comment by martonsz** [comment]
I created a PR for this https://github.com/coollabsio/coolify/pull/2719

**Comment by peaklabs-dev** [comment]
This will be released in one of the next versions

**Comment by jooles** [comment]
@peaklabs-dev Any ETA? 😅

btw, you're awesome, man. What you have already done and accomplished in such a short time... huge respect!

**Reply by peaklabs-dev** [reply]
Thanks for the kind words. I cannot give you an ETA - it will be released when it is ready. If you want to deploy it now, you can use the Docker Compose deploy type with the following template: https://github.com/coollabsio/coolify/blob/next/templates%2Fcompose%2Fpaperless.yaml

**Reply by jooles** [reply]
Thank you very much for that, that really helps! 🙏
